src/Chapter13/Table13.01.a.LambdaExpressionNotesAndExamples.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Justification: Only snippets of source code shown for elucidation.
+#pragma warning disable CS0168 // Variable is declared but never used
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Table13_01;
+
+public partial class LambdaExpressionNotesAndExamples
+{
+    // 1.
+    static public void DiscardParameters()
+    {
+#if !NET6_0_OR_GREATER
+        Action<int, int> x = (_,_)=>
```

Comment:
nit: We generally add a space after commas
```suggestion
        Action<int, int> x = (_, _)=>
```